pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarRecordCursor.java

```diff
@@ -91,6 +92,7 @@
     // are empty or not
     private final long splitSize;
     private long entriesProcessed = 0;
+    private int partition = -1;
```

Comment:
For non-partitioned topics, should the partition column be NULL?  Currently, for internal columns, if there is not an applicable value, the value is displayed as "NULL"

Reply:
Is it necessary to be consistent with the messageId? If not, agree use NULL for non-partitioned topics.

Reply:
personally I am in favor of using -1 to be consistent with how we use it in MessageId and it make a consistent view for both partitioned topic and non-partitioned topic.

A partitioned topic has N partitions, a non-partitioned topic has one partition with partition id "-1".

Reply:
I am ok with either.  @merlimat any opinion?

Reply:
after discussing with @KarthikRamasamy since he has experience with databases, it seems that "null" can have all sort of meanings.  Perhaps its just better to use -1 for non-partitioned topics